tests/PHPUnit/Unit/Scheduler/TimetableTest.php

```diff
@@ -61,6 +63,19 @@ public function testGetTimetableFromOptionValue($expectedTimetable, $option)
         self::resetPiwikOption();
     }
 
+    public function testRescheduleTaskAndRunTomorrow()
+    {
+        $timetable = new Timetable();
```

Comment:
No. The test class extends `\PHPUnit_Framework_TestCase`. It would need to be an Integration or System test to have a database setup.
Nevertheless the test might work if another test using database ran before, but that's not always the case.